### DISCUSSION TITLE
Components Theming - Additional tokens

### DISCUSSION THREAD

**Discussion by jagadeeshaby** [cloudscape-design/components#217] [Q&A] [answered]
Looks like only few of the design tokens are exposed for theming (https://cloudscape.design/foundation/visual-foundation/design-tokens/#tokens), i would like to understand if we can configure all the below variables. 

https://github.com/cloudscape-design/components/blob/d74774c591ec02ddb4938859159cf6227cb8c648/src/__integ__/__snapshots__/themes.test.ts.snap

For now i would like to update the border radius value set on buttons and alert components on my themed version. please help.

**Comment by johannes-weber** [comment, marked as answer]
Initially we bound theming to adopt the branding of the Design System. We exposed groups of tokens for theming instead of individual tokens for consistency.
Since then we are exposing more tokens based on customer demand and their use case. Exposing a new design token for theming is basically a change in our API. Before we do so, we want to learn based on the customers' use cases and act accordingly.

We have this #221 PR about making the border-radius design tokens public and themeable. I recommend observing the PR to get notified when it gets merged.

**Reply by rikkuness** [reply]
One that I've found I'm missing is access to configure the color of `shadow-container`, I can use theming to change for example `color-background-container-content` but whatever I change the container background to, there's always a dark blue box shadow around the outside

**Comment by fralongo** [comment]
@jagadeeshaby We have just released public and themeable design tokens for border radius. In our documentation website you can find:
- The [list of border radius tokens](https://cloudscape.design/foundation/visual-foundation/design-tokens/#tokens).
- Additional [implementation guidelines for theming](https://cloudscape.design/foundation/visual-foundation/theming/#implementation).
